nft_ingester/src/program_transformers/bubblegum/mint_v1.rs

```diff
@@ -204,6 +202,8 @@ where
                     return Ok(None);
                 }
 
+                multi_txn.commit().await?;
+
```

Comment:
This needs to be above the URI check or it will return before committing all the other info.